cv32/sim/uvmt_cv32/waves.tcl

```diff
@@ -0,0 +1,74 @@
+# catch {
```

Comment:
Same comment: I would prefer not to have tool-specific Tcl commands files in our repo.